projects/observability/src/shared/components/cartesian/d3/legend/cartesian-legend.ts

```diff
@@ -50,6 +70,20 @@ export class CartesianLegend {
     this.summaryControl && this.summaryControl.destroy();
   }
 
+  private drawReset(container: ContainerElement): void {
+    select(container)
+      .append('span')
+      .classed(CartesianLegend.RESET_CSS_CLASS, true)
+      .text('Reset')
+      .on('click', () => this.resetToDefault());
+
+    this.toggleReset(this.isDefault);
+  }
+
+  private toggleReset(isHidden: boolean): void {
```

Comment:
nit: toggle implies we're flipping something without an arg, but since we'r'e providing the input here, it's just a set function.

Reply:
changed it